site/docs/guide/commands.md

```diff
@@ -75,3 +75,6 @@ When a user clicks on the button underneath the post, their Telegram client will
 This way, your bot can identify where a user came from, and that they clicked the button underneath a specific channel post.
 
 Naturally, you can also embed such links anywhere else: on the web, in messages, in QR codes, etc.
+
+Check out this [section of the Telegram docs](https://core.telegram.org/api/links#bot-links) to see a full list of possible link formats.
```

Comment:
```suggestion
Check out [this section of the Telegram docs](https://core.telegram.org/api/links#bot-links) to see a full list of possible link formats.
```

Reply:
Didn't we agree to never include __the__ in inks? I assumed this holds for __this__, __that__, __these__, and __those__, too.

Reply:
Right.